cmd/installer/cli/restore2.go

```diff
@@ -92,7 +92,7 @@ func Restore2Cmd(ctx context.Context, name string) *cobra.Command {
 		Use:   "restore",
 		Short: fmt.Sprintf("Restore a %s cluster", name),
 		PreRunE: func(cmd *cobra.Command, args []string) error {
-			if err := preRunInstall2(cmd, &flags); err != nil {
+			if err := preRunInstall2(cmd, &flags, false); err != nil {
```

Comment:
is the license not required to restore?

Reply:
I don't think so? the license is stored inside the admin console after all

Reply:
sure but what about custom domains? why not just make it required?

Reply:
because the only thing we'd need it for is metrics and it feels wrong to require it for just metrics
easy to change later of course